corehq/apps/export/forms.py

```diff
@@ -1055,6 +1042,20 @@ def get_edit_url(self, export):
         return reverse(EditNewCustomFormExportView.urlname,
                        args=(self.domain_object.name, export._id))
 
+    def _get_es_user_types(self, mobile_user_and_groups_slugs=None):
+        """
+        Return a list of elastic search user types (each item in the return list
+        is in corehq.pillows.utils.USER_TYPES) corresponding to the selected
+        export user types.
+        """
+        es_user_types = []
+        export_user_types = self._get_selected_user_types(mobile_user_and_groups_slugs)
+        export_to_es_user_types_map = self._EXPORT_TO_ES_USER_TYPES_MAP
```

Comment:
should this class attr also be moved along?  Also, this is a strange line of code.

Reply:
Inlined this `_EXPORT_TO_ES_USER_TYPES_MAP` in 8f2b8d0d02 along with a weird test that seems to just verify its value. There is indeed a lot of indirection in this code, there's probably more that could be cleaned up.